library/database/class.dataset.php

```diff
@@ -31,35 +31,38 @@ class Gdn_DataSet implements IteratorAggregate, Countable, JsonSerializable {
     public $Connection;
 
     /** @var int The index of the $this->_ResultSet currently being accessed. */
-    private $_Cursor = -1;
+    private $cursor = -1;
 
     /**
-     * @var intDetermines what type of result is returned from the various methods by default.
+     * @var string Determines what type of result is returned from the various methods by default.
      * Either DATASET_TYPE_OBJECT or DATASET_TYPE_ARRAY.
      */
     protected $_DatasetType = DATASET_TYPE_OBJECT;
 
     /** @var bool  */
-    protected $_EOF = false;
+    protected $eof = false;
```

Comment:
Renaming the private properties is A-OK, but renaming the protected property is a little riskier. Are you confident in this change?

Reply:
Sounds like we're confident it's very likely anything out there is subclassing `Gdn_DataSet`, so this is not a concern.